src/components/Modals/FiatRamps/config.ts

```diff
@@ -64,6 +91,9 @@ export const supportedFiatRamps: SupportedFiatRamp = {
       const sellAssetIds = parseGemSellAssets(currencyList)
       return [buyAssetIds, sellAssetIds]
     },
+    getSupportedFiatList: () => {
+      return getSupportedGemFiatCurrencies()
+    },
```

Comment:
```suggestion
    getSupportedFiatList: getSupportedGemFiatCurrencies,
```

we can do this for all of them for consistency if you like, not going to block on it though